frontend/src/routes/Settings.tsx

```diff
@@ -4,203 +4,247 @@ import MutinyToaster from "@components/MutinyToaster";
-                                <p className="text-2xl font-light">Node Pubkey</p>
-                                <div className="flex items-center gap-4">
-                                    <pre>
-                                        <code>{takeN(nodes[0], 25)}</code>
-                                    </pre>
-                                    <Copy copyValue={nodes[0]} />
-                                </div>
-                            </> : <button onClick={handleNewNode}>New Node</button>
-                        }
-                    </div>
-                    {COMMIT_HASH &&
-                        <div>
-                            <p className="text-2xl font-light">Mutiny Version</p>
-                            <div className="flex items-center gap-4">
-                                <pre>
-                                    <code>{COMMIT_HASH}</code>
-                                </pre>
-                                <Copy copyValue={COMMIT_HASH} />
-                            </div>
-                        </div>
-                    }
-                    <SettingStringsEditor />
-                    <div />
-                    <div />
-                    <div />
-                    <div />
-
-                    <div className="bg-red p-4 rounded w-full">
-                        <p className="text-2xl font-light text-white uppercase">Danger Zone</p>
-                    </div>
-
-                    <button onClick={handleClearState}>Clear State</button>
-
-                    <button onClick={handleSave}>Save State As File</button>
-
-                    <label className="fileInputButton">Upload Saved State
-                        <input className="fileInput" type="file" onChange={handleFileChoose} />
-                    </label>
+    setDialogOpen(false);
+  }
+
+  async function handleNewNode() {
+    try {
+      await nodeManager?.new_node();
+      await queryClient.invalidateQueries({ queryKey: ["nodes"] });
+    } catch (e) {
+      console.error(e);
+      toastAnything(e);
+    }
+  }
+
+  return (
+    <>
+      <header className="px-8 pt-8 flex justify-between items-center">
+        <PageTitle title="Settings" theme="red" />
+        <Close />
+      </header>
+      <main className={mainWrapperStyle({ padSides: "no" })}>
+        <div className="flex flex-col gap-4 flex-1 overflow-y-scroll px-8 pb-[12rem] items-start">
+          <div>
+            <p className="text-2xl font-light">
+              Write down these words or you'll die!
+            </p>
+            <SeedWords words={words ?? ""} />
+          </div>
+          <div className="w-full">
+            {nodes && nodes[0] ? (
+              <>
+                <p className="text-2xl font-light">Node Pubkey</p>
+                <div className="flex items-center gap-4 w-full">
+                  <pre className="w-full truncate md:w-max">
+                        {nodes[0]}
+                  </pre>
```

Comment:
I figure this class based change can be implemented throughout the codebase in combination with some strings retaining the `takeN` util function